modules/ztp-sno-du-removing-the-console-operator.adoc

```diff
@@ -6,6 +6,14 @@
 [id="ztp-sno-du-removing-the-console-operator_{context}"]
 = Console Operator
 
-The console-operator installs and maintains the web console on a cluster. When the node is centrally managed the Operator is not needed and makes space for application workloads. The following `Console` custom resource (CR) example disables the console.
+The Console Operator installs and maintains the web console on a cluster.
```

Comment:
In 4.13 this CR is no longer needed (or recommended). The reference SiteConfig uses installConfigOverrides to remove the Console operator entirely (composable openshift).

Reply:
Yes this is updated. 

```
To disable the Console Operator during the installation of the managed cluster, set the following in the `spec.clusters.0.installConfigOverrides` field of the `SiteConfig` custom resource (CR):

[source,yaml]
----
installConfigOverrides:  "{\"capabilities\":{\"baselineCapabilitySet\": \"None\" }}"
----
```

Reply:
(Also reworded the text slightly.)